#SV                        : Satellite Views (int)     
#HDOP                      : Horizontal Dilution of Precision (float) 
#VDOP                      : Vertical Dilution of Precision (float)  
#Latitude                  : Latitude (Degrees)
#Longitude                 : Longitude (Degrees) 
#Altitude                  : Altitude (meters)
#Status                    : GPS Status

uint8 SV
float32 HDOP
float32 VDOP
float32 Latitude
float32 Longitude
float32 Altitude
int8 Status
